js/apps/account-ui/src/oid4vci/Oid4Vci.tsx

```diff
@@ -51,6 +51,7 @@ export const Oid4Vci = () => {
 
   useEffect(() => {
     if (initialSelected !== selected && credentialsIssuer !== undefined) {
+      // eslint-disable-next-line
```

Comment:
Instead of disabling all rules for the next line, it would be better to disable the specific rule.
```suggestion
      // eslint-disable-next-line @typescript-eslint/no-floating-promises
```